nncf/quantization/algorithms/min_max/algorithm.py

```diff
@@ -204,6 +209,7 @@ def __init__(
         self._preset = preset
         self._ignored_scope = IgnoredScope() if ignored_scope is None else ignored_scope
         self.quantizer_propagation_rule = quantizer_propagation_rule
+        self.reducer_map = REDUCERS_MAP
```

Comment:
Please introduce a `reducers_map` property in the `MinMaxAlgoBackend` class and do not introduce any attributes in the  `MinMaxQuantization` class

Reply:
Yes, my bad. On it.